uefi/Cargo.toml

```diff
@@ -12,19 +12,22 @@ license = "MPL-2.0"
 rust-version = "1.68"
 
 [features]
-default = ["panic-on-logger-errors"]
+default = ["panic-on-logger-errors", "draw_target"]
 alloc = []
 global_allocator = []
 logger = []
 # Ignore text output errors in logger as a workaround for firmware issues that
 # were observed on the VirtualBox UEFI implementation (see uefi-rs#121).
 # In those cases, this feature can be excluded by removing the default features.
 panic-on-logger-errors = []
+# DrawTarget for GOP protocol
+draw_target = ["embedded-graphics-core"]
 # Generic gate to code that uses unstable features of Rust. You usually need a nightly toolchain.
 unstable = []
 
 [dependencies]
 bitflags = "2.0.0"
+embedded-graphics-core = { version = "0.4.0", optional = true }
```

Comment:
I think that we do not need a feature flag for this.

Reply:
I don't think that this will end up in a binary, if the GOP is not used.

Reply:
something something rust zero cost abstraction yada yada? :D 

more seriously: I was reasoning on a purely "developer" side, bringing more dependency is more annoying usually, but this might be a fair tradeoff here given it's tailored for embedded systems.